Cannot find module 'antd/locale/xx_XX'

### Reproduction link

[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/type-antd-5-1-3-forked-pd0w4u?file=/demo.tsx)

### Steps to reproduce

Just try to compile the project

### What is expected?

should import locale settings

### What is actually happening?

Cannot find module 'antd/locale/xx_XX

| Environment | Info |
| --- | --- |
| antd | 5.1.3 |
| React | React ^18.2.0 |
| System | MacOS and CodeSandbox |
| Browser | Chrome 108.0.5359.124 |

<!-- generated by ant-design-issue-helper. DO NOT REMOVE -->

--- a/package.json
+++ b/package.json
@@ -155,7 +155,7 @@
     "throttle-debounce": "^5.0.0"
   },
   "devDependencies": {
-    "@ant-design/tools": "^17.0.0-alpha.5",
+    "@ant-design/tools": "^17.0.0-alpha.7",
     "@babel/eslint-plugin": "^7.19.1",
     "@emotion/babel-preset-css-prop": "^11.10.0",
     "@emotion/css": "^11.10.5",

--- a/scripts/post-script.js
+++ b/scripts/post-script.js
@@ -27,6 +27,7 @@ const DEPRECIATED_VERSION = {
   '5.0.6': ['https://github.com/ant-design/ant-design/issues/39807'],
   '5.1.0': ['https://github.com/react-component/drawer/pull/370'],
   '5.1.2': ['https://github.com/ant-design/ant-design/issues/39949'],
+  '5.1.3': ['https://github.com/ant-design/ant-design/issues/40113'],
 };
 
 function matchDeprecated(version) {

